client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -81,6 +81,16 @@
           </span>
         </div>
 
+        <div :class="prefixClass('c-statement__formhint flash-info u-mb-0_5')">
```

Comment:
I'd suggest to move this text into the blue box that is already there:
![statement_modal_hint](https://github.com/user-attachments/assets/4caec539-8211-41b0-ad8c-0247976e4366)

That way, the error box is displayed below the blue box and not between two blue boxes (which I'd find a little chaotic).

Also, the icon on the left is not properly aligned with the text, so I'd adjust the `__hint-icon` class in statement.scss:
```
&__hint-icon {
  position: absolute;

  line-height: 1.25;
}
```
I am also fine with doing this myself and requesting another FE dev for review. :)

Reply:
hey @hwiem yes, i think that might be the best. Then i will assign the pr to you ;)